pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "support-docs"
+version = "0.1.0"
+description = ""
+authors = ["SecureDrop team <securedrop@freedom.press>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+# The RTD theme does not yet support Sphinx >=6.0.0.
+# See https://github.com/readthedocs/sphinx_rtd_theme/issues/1323
+sphinx = "<6.0.0"
+sphinx-autobuild = "^2021.3.14"
+sphinx-rtd-theme = "^1.1.1"
+
+[build-system]
```

Comment:
Given that this is not an installable Python package, do we need this part?

Reply:
Nope!